trading_calendars/us_holidays.py

```diff
@@ -28,7 +28,7 @@
 
 
 def july_5th_holiday_observance(datetime_index):
-    return datetime_index[datetime_index.year != 2013]
+    return datetime_index[datetime_index.year not in (2013, 2019)]
```

Comment:
Hmm, this errors for me - does it work for you? Perhaps we need something like `numpy.isin` but which supports numpy>=1.11.

Reply:
Yep - that was the bug. Fixed in https://github.com/quantopian/trading_calendars/pull/57/commits/32ef5009ae1af178df61b9d4c95101d45cb65020

Reply:
Ah I see what you meant - there's still an issue in 1.11.3

Reply:
Trying again in https://github.com/quantopian/trading_calendars/pull/57/commits/951a91150e7e7fd1a7cfc509723dd4a61531bd93